cpu/sam0_common/periph/rtt.c

```diff
@@ -0,0 +1,212 @@
+/*
+ * Copyright (C) 2015 Kaspar Schleiser <kaspar@schleiser.de>
+ *               2015 FreshTemp, LLC.
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     cpu_sam0_common
+ * @ingroup     drivers_periph_rtt
+ * @{
+ *
+ * @file        rtt.c
+ * @brief       Low-level RTT driver implementation
+ *
+ * @author      Kaspar Schleiser <kaspar@schleiser.de>
+ *
+ * @}
+ */
+
+#include <stdint.h>
+#include "periph/rtt.h"
+#include "board.h"
+
+#define ENABLE_DEBUG 0
+#include "debug.h"
+
+static rtt_cb_t _overflow_cb;
+static void* _overflow_arg;
+
+static rtt_cb_t _cmp0_cb;
+static void* _cmp0_arg;
+
+static void _wait_syncbusy(void)
+{
+#ifdef REG_RTC_MODE2_SYNCBUSY
```

Comment:
Just notice this !
```suggestion
#ifdef REG_RTC_MODE0_SYNCBUSY
```
Otherwise we should be good ! Could you squash and rebase please ? I'll run one last test before go right after.

Reply:
Done.